local.properties

```diff
@@ -0,0 +1,8 @@
+## This file must *NOT* be checked into Version Control Systems,
```

Comment:
This file should never be pushed, it is environment specific and could break the local build of following pulls depending on the host